### ISSUE TITLE
[@angular-eslint/prefer-output-readonly] ignore prefer output readonly if there is reassignment in code

### ISSUE THREAD

**Issue by splincode** [angular-eslint/angular-eslint#2673]
There is a reassignment in code

```ts
public quiet: OutputRef<unknown> = output();


protected init(): void {
  this.quiet = outputFromObservable(
      interval(POLLING_TIME)
  );
}
```

But eslint throw error 

```
ESLint: Prefer to declare `OutputEmitterRef` as `readonly` since they are not supposed to be reassigned (@angular-eslint/prefer-output-readonly)
ESLint: Prefer to declare `OutputRef` as `readonly` since they are not supposed to be reassigned (@angular-eslint/prefer-output-readonly)
```

**Comment by JamesHenry** [comment]
@splincode What are you reporting here? The rule is doing exactly what it says?

**Comment by splincode** [comment]
But I need reassign here, if I explicit reassign it should be ignore

**Comment by JamesHenry** [comment]
That doesn't really make sense I'm afraid, the rule's whole purpose is very specifically about recommending against what you are doing and can't adapt itself to what you feel you want to do in a situation. That's like saying `const` should be fine with you re-assigning a variable if you _have to_. No, if you have to reassign you change your code, you use `let` instead. 

Here the equivalent is that you would suppress the lint rule - you are knowingly and consciously going against its guidance, so you want to shut it off (e.g. with an `// eslint-disable-next-line @angular-eslint/prefer-output-readonly`). Disable comments are ok, it's your code at the end of the day.

It's possible there is a different refactor you could make to keep the rule happy and still have things behave how you want, but it's such a small code snippet without any context so it's impossible to propose anything else other than suppressing the rule.

Does that make sense?

**Comment by splincode** [comment]
Thank you
